_maps/configs/pgf_woeful_cthonian.json

```diff
@@ -2,6 +2,7 @@
 	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
 	"faction": "/datum/faction/pgf",
 	"prefix": "PGFN",
+	"manufacturer": "Etherbor Lihmona Shipyards",
```

Comment:
technically fluff says - "manufactured by Axilai Orbital Works".

Reply:
Guh I knew the one PGF ship that I assumed would be Etherbor would end up not being that..